### PR TITLE
[QUEUES] Minor syntax changes

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**
PCX-8036

### CODE REVIEW

File: content/queues/examples/list-messages-from-dash/index.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ meta:
 
 ## List messages from the dashboard
 
-Listing messages from the dashboard allows you to debug Queues or Queue producers without a consumer Worker. Fetching a batch of messages to preview will not acknowledge or retry the message or affect its position in the Queue. The Queue can still be consumed normally by a consumer Worker.
+Listing messages from the dashboard allows you to debug Queues or Queue producers without a consumer Worker. Fetching a batch of messages to preview will not acknowledge or retry the message or affect its position in the queue. The Queue can still be consumed normally by a consumer Worker.
```
Comment (marciocloudflare) [inline content/queues/examples/list-messages-from-dash/index.md]: Should `The Queue can still be consumed...` be also lowercased? Seems to be talking about a queue, not the product.

File: content/queues/examples/send-messages-from-dash.md [hunk 2]
```diff
@@ -10,18 +10,18 @@ meta:
 
 # Send messages from the dashboard
 
-Sending messages from the dashboard allows you to debug Queues or Queue consumers without a producer Worker.
+Sending messages from the dashboard allows you to debug Queues or queue consumers without a producer Worker.
```
Comment (marciocloudflare) [inline content/queues/examples/send-messages-from-dash.md:13]: The first `Queue` here is the product, right?

**Review by marciocloudflare — APPROVED** [review]

File: content/queues/examples/list-messages-from-dash/index.md (hunk 1 above)
Comment (Maddy-Cloudflare) [reply content/queues/examples/list-messages-from-dash/index.md]: Yes, it should. Will fix that.

File: content/queues/examples/send-messages-from-dash.md (hunk 2 above)
Comment (Maddy-Cloudflare) [reply content/queues/examples/send-messages-from-dash.md:13]: Yes, the first one is the product.

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**